todo/src/components/App.js

```diff
@@ -0,0 +1,87 @@
+import React, { Component } from 'react';
+import './App.css';
+import { addTodo, toggleComplete, actionDeleteTodo } from '../actions/actions';
+import { connect } from 'react-redux';
+
+class App extends Component {
+  state = {
+    inputText:''
+  }
+
+  handleChange = ev => {
+    this.setState({ [ev.target.name]: ev.target.value});
+  }
+
+  handleSubmit = ev => {
+    ev.preventDefault();
+    console.log('clicked');
+    this.props.addTodo(this.state.inputText);
+    this.setState({
+      inputText: ''
+    })
+  }
+
+  toggleTodo = index => {
+    this.props.toggleComplete(index);
+  }
+
+  deleteTodo = index =>{
+    this.props.actionDeleteTodo(index);
+  }
```

Comment:
It's a little curious, to have class methods just to call methods off of props. Why not just use the props methods directly?